src/components/MessageList/MessageListStyled.js

```diff
@@ -1,3 +1,6 @@
 import styled from 'styled-components';
 
-export const MessageListStyled = styled.ul``;
+export const MessageListStyled = styled.ul`
+  //   display: flex;
+  //   flex-direction: column-reverse;
```

Comment:
Should it be commented? It seems like you added styles and forgot uncommented it or you moved it into the ChatStyled.js and forgot to deleted it here.

Reply:
that's right, thank you